producers/kubernetes/ingress.go

```diff
@@ -57,8 +57,8 @@ func (a *kubernetesIngressProducer) Endpoints() ([]*pkg.Endpoint, error) {
 
 		eps, err := a.convertIngressToEndpoint(ing)
```

Comment:
as a side note, ingress having more than two loadbalancers will result in validation fail, hence DNS record being removed :(